include/verilatedos.h

```diff
@@ -475,6 +475,8 @@ using ssize_t = uint32_t;  ///< signed size_t; returned from read()
 #  define VL_CPU_RELAX() asm volatile("yield" ::: "memory")
 # elif defined(__powerpc64__)
 #  define VL_CPU_RELAX() asm volatile("or 1, 1, 1; or 2, 2, 2;" ::: "memory")
+# elif defined(__loongarch__)
+#  define VL_CPU_RELAX() asm volatile("nop" ::: "memory")
```

Comment:
```suggestion
# elif defined(__loongarch__)
// LoongArch does not currently have a yield/pause instruction
#  define VL_CPU_RELAX() asm volatile("nop" ::: "memory")
```

You're right, I looked at their instruction documentation.  It's odd they didn't define it anyways but implement it as a NOP for now, to provide forward compatibility.  I'd just like a comment so this isn't copied by other architectures by mistake.

Reply:
Thank you. Can this PR be merged to upstream ? 😊